### ISSUE TITLE
cross entrophy loss 

### ISSUE THREAD

**Issue by wenfahu** [KaiyangZhou/pytorch-center-loss#1]
I'm afraid that u're writing the wrong code for the classification loss. In pytorch, the softmax cross entrophy loss should be 

**NLL + LogSoftmax**

or

**nn.CrossEntropyLoss + logits**

**Comment by KaiyangZhou** [comment]
fixed
